lib/mix/tasks/gen.user_from_csv.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Mix.Tasks.Gen.UserFromCsv do
+  use Mix.Task
+  alias Ecto.Multi
+  alias Safira.Repo
+  alias Safira.Auth
+  alias Safira.Accounts.User
+  alias Safira.Accounts.Attendee
+
+  NimbleCSV.define(SeiParser, separator: ",", escape: "\"")
+
+  def run(args) do
+    cond do
+      Enum.empty?(args) ->
+        Mix.shell().info("Needs to receive a file URL.")
+
+      true ->
+        args |> List.first() |> create
+    end
+  end
+
+  defp create(path) do
+    Mix.Task.run("app.start")
+
+    if File.exists?(path) do
+      path
+      |> parse_csv
+      |> create_users
+    else
+      IO.puts("File does not exist")
+    end
+  end
+
+  defp parse_csv(path) do
+    path
+    |> File.stream!(read_ahead: 1_000)
+    |> SeiParser.parse_stream()
+    |> Stream.map(fn row ->
```

Comment:
Prefer pattern matching here to later using `Enum.at/2`

Reply:
Thing is, the csv has, 10+ different collumns, since we only 3 of those values I found it a bit unnecessary to use pattern matching as it will result in a quite long line of code.